x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/index.tsx

```diff
@@ -343,6 +356,50 @@ export const AddExceptionModal = memo(function AddExceptionModal({
     return false;
   }, [maybeRule]);
 
+  const OsOptions: Array<EuiComboBoxOptionOption<OsType>> = useMemo((): Array<
+    EuiComboBoxOptionOption<OsType>
+  > => {
+    return [
+      {
+        label: 'Windows',
+        value: 'windows',
+      },
+      {
+        label: 'macOS',
+        value: 'macos',
+      },
+      {
+        label: 'Linux',
+        value: 'linux',
+      },
+    ];
+  }, []);
+
+  const handleOSSelectionChange = useCallback(
+    (selectedOptions): void => {
+      setSelectedOs(selectedOptions[0].value);
+    },
+    [setSelectedOs]
+  );
+
+  const selectedOStoOptions = useMemo((): Array<EuiComboBoxOptionOption<OsType>> => {
```

Comment:
I had a question kind of orthogonal to @paul-tavares , I know our previous logic for the non-alert-generated exceptions was to enrich the OS type to all possible types, but this looks like we only have the ability to select one, even though you're using a `ComboBox` instead of a `SuperSelect`. It's kind of overwriting the main use case of a combo box. If we aren't keeping the ability to select multiple OS's why is that? and if we aren't, we might wanna use a `SuperSelect` like paul had suggested

Reply:
> If we aren't keeping the ability to select multiple OS's why is that?

Linux Exceptions and Mac/Windows Exceptions have fields that should NOT overlap, i.e. `file.path.caseless`.  Since this is the case, we need users to be intentional about the OS they select.

I'll go back and update this stuff to use `SuperSelect` since that's the appropriate component

Reply:
@paul-tavares @dplumlee update re: `SuperSelect`

I went in and changed it out locally, the biggest problem I see with it is that it doesn't seem to honor the `placeholder` attribute and so doesn't seem like the best option when I want to intentionally leave the OS selection blank at first

Using `SuperSelect`
![image](https://user-images.githubusercontent.com/56395104/116310420-fd723800-a777-11eb-8ffd-46c715350762.png)

Using `ComboBox`
![image](https://user-images.githubusercontent.com/56395104/116310610-3dd1b600-a778-11eb-8a80-240b38f8c354.png)

In my mind, the UX is much better with the `placeholder` and I think it warrants that I should use `ComboBox`.  Thoughts?

Reply:
@kevinlog in regards to the `ComboBox`/`SuperSelect`, I'd agree that the placeholder looks better and it keeps the door open for adding back the logic of multi-select further down the road. I mainly brought it up because I knew there was an existing component that did the same thing you were using `ComboBox` for. I'll give the +1 for using it here

Reply:
> Linux Exceptions and Mac/Windows Exceptions have fields that should NOT overlap, i.e. file.path.caseless. Since this is the case, we need users to be intentional about the OS they select.
@kevinlog I think Davis' point is a valid one in the cases where users may want to add an exception that applies to both Mac and Windows. Currently, when an user adds an Endpoint exception from scratch, it gets applied to both OSes. Of course, the current behavior has pros and cons, but with this update users who want to add an exception to both Windows and Mac will have go through the workflow twice. I'm not suggesting we necessarily make any changes in this PR but it is something we should consider.

Reply:
@dplumlee @peluja1012 thanks for following up and the input.

re: allowing multi OS select.  I understand the use case and you both certainly have valid points - I'm wondering what the best UX would be for the form.  We'd need to enforce conditions that the user can pick (Mac AND/OR Windows) OR (Linux) because Linux should remain on its own due to the `file.path` limitations.  I'll create a ticket for this case and get further input

Reply:
@kevinlog Thanks for that. I think there are a few ways to differentiate between the two with native `ComboBox` [options like groups or something](https://elastic.github.io/eui/#/forms/combo-box). That could be a start from a UX perspective and we could add our own logic too, perhaps with the selection ability itself